middleware/static_windows.go

```diff
@@ -0,0 +1,23 @@
+package middleware
+
+import (
+	"os"
+)
+
+// We ignore these errors as there could be handler that matches request path.
+//
+// As of 1.20 on Windows filepath.Clean has different behaviour on OS related filesystems so we need to use path.Clean
+// which is more suitable for path coming from web but this has some caveats on Windows. When we eventually end up in
+// os related filesystem Open methods we are getting different errors as earlier versions. As of 1.20 path checks are
+// more strict on path you provide and consider path with [UNC](https://en.wikipedia.org/wiki/Path_(computing)#UNC)
+// but missing host etc parts as invalid. Previously it would result you `fs.ErrNotExist`.
+//
+// So for 1.20@Windows we need to consider it as same not exist so we can continue next middleware/handler and not error
+// which would result status 500 instead of potential route hit or 404.
+func isIgnorableOpenFileError(err error) bool {
```

Comment:
Suggestion:

// As of Go 1.20 filepath.Clean has different behaviour on OS related filesystems so we need to use path.Clean 
// on Windows which has some caveats. The Open methods might return different errors than earlier versions and 
// as of 1.20 path checks are more strict on the provided path and considers [UNC](https://en.wikipedia.org/wiki/Path_(computing)#UNC)
// paths with missing host etc parts as invalid. Previously it would result you `fs.ErrNotExist`.
//
// For 1.20@Windows we need to treat those errors the same as `fs.ErrNotExists` so we can continue handling
// errors in the middleware/handler chain. Otherwise we might end up with status 500 instead of finding a route 
// or return 404 not found.